src/Faker/Caching/PropertyHelper.cs

```diff
@@ -13,6 +13,12 @@ namespace Faker.Caching
 	/// </remarks>
 	internal static class PropertyHelper
 	{
+		/// <summary>
+		///   Extracts the PropertyInfo for the member selector
```

Comment:
This I believe should be changed to
`Extracts the Property Information for the member selector`